src/cryptenroll/cryptenroll-tpm2.c

```diff
@@ -142,7 +142,8 @@ int enroll_tpm2(struct crypt_device *cd,
         _cleanup_(erase_and_freep) void *secret = NULL;
         _cleanup_(json_variant_unrefp) JsonVariant *v = NULL, *signature_json = NULL;
         _cleanup_(erase_and_freep) char *base64_encoded = NULL;
-        size_t secret_size, blob_size, hash_size, pubkey_size = 0;
+        _cleanup_(freep) void *srk_buf = NULL;
```

Comment:
doesn't matter, but free() is so fundamental, so we actually do grant it a macro of its own: `_cleanup_free_`...

but it's a weird rule, one we probably should get rid of, so maybe ignore this comment.